lib/php/rendition-factory/src/Format/FormatInterface.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Alchemy\RenditionFactory\Transformer\Video\FFMpeg\Format;
```

Comment:
It should stay into `Alchemy\RenditionFactory\Transformer\Video\FFMpeg` namespace as it's only intended to FFMPEG

Reply:
the idea is to use "formats" on all modules, allowing common options / code for `format`, `extension`

Reply:
The aim of your Format classes was to provide the FFMPEG one via `getFFMpegFormat` method.
Without this method, having classes has no interest IMO. And it is not used anywhere else than in FFMPEG transformation modules.